cypress/e2e/cloud/annotations.test.ts

```diff
@@ -254,14 +254,9 @@ describe('The Annotations UI functionality', () => {
         .click()
         .then(() => {
           cy.getByTestID('selector-list schmucket').click()
-          cy.getByTestID(`selector-list m`)
-            .should('exist')
-            .click()
-          cy.getByTestID('selector-list v')
-            .should('exist')
-            .click()
+          cy.getByTestID(`selector-list m`).click()
+          cy.getByTestID('selector-list v').click()
```

Comment:
let's try it. but i think we've [already tried this solution.](https://github.com/influxdata/ui/pull/1312) Gene said he ran it locally a bunch and didn't get any flakiness, so let's see how this goes. 🤞 

cc @genehynson

Reply:
yeah i was reading that cypress 6+ suggests you remove the .should chaining, im not sure if it will actually fix anything. Also i got kind of excited when the CI test failed, till i saw it was firefox, on the login test file -_-